src/framework/mpas_dmpar.F

```diff
@@ -5406,17 +5416,17 @@ subroutine mpas_dmpar_exch_halo_field1d_real(field, haloLayersIn)!{{{
 
    end subroutine mpas_dmpar_exch_halo_field1d_real!}}}
 
-!-----------------------------------------------------------------------
-!  routine mpas_dmpar_exch_halo_field2d_real
-!
-!> \brief MPAS dmpar halo exchange 2D real field
-!> \author Doug Jacobsen
-!> \date   03/26/13
-!> \details
-!>  This routine handles the halo exchange communication of an input field across all processors.
-!>  It requires exchange lists to be created prior to calling this routine.
-!
-!-----------------------------------------------------------------------
+   !-----------------------------------------------------------------------
+   !  routine mpas_dmpar_exch_halo_field2d_real
+   !
+   !> \brief MPAS dmpar halo exchange 2D real field
+   !> \author Doug Jacobsen
+   !> \date   03/26/13
+   !> \details
+   !>  This routine handles the halo exchange communication of an input field across all processors.
+   !>  It requires exchange lists to be created prior to calling this routine.
+   !
+   !-----------------------------------------------------------------------
```

Comment:
Although I would agree that it would have been better for the comment block to have the same indentation as the `subroutine` statement, I would suggest we don't change whitespace here just for this function. In future, we could make a separate PR just for the purpose of fixing indentation consistently throughout the module.